src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/loadingScripts.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
The name of the file is a little bit off to me. Every file in JS contains scripts, so I would drop the suffix "Scripts" here as well